catalog/spatial/csw/spatial-csw-endpoint/src/main/java/org/codice/ddf/spatial/ogc/csw/catalog/endpoint/CswEndpoint.java

```diff
@@ -1124,6 +1133,41 @@ private FilterCapabilities buildFilterCapabilities() {
     return filterCapabilities;
   }
 
+  private Set<String> getQueryFilterTransformerTypeNames() {
+    return Stream.of(getBundle())
```

Comment:
✏️ This feels strange to me. I'm not sure why you wouldn't just push the calls for `getBundle().getBundleContext()` down to `getQueryFilterTransformerServices` and change that method to return a `Stream`.

Reply:
Either approach is valid. I find my impl more natural because a start with a bundle, turn it into a bundle context, and then turn the bundle context into a collection of services.